geth/common/rpccall.go

```diff
@@ -105,12 +105,13 @@ func (r RPCCall) ParseGas() *hexutil.Big {
 		return nil
 	}
 
-	parsedValue, err := hexutil.DecodeBig(inputValue)
+	parsedValue, err := hexutil.DecodeUint64(inputValue)
 	if err != nil {
 		return nil
 	}
 
-	return (*hexutil.Big)(parsedValue)
+	_v := hexutil.Uint64(parsedValue)
```

Comment:
`_v` is a bit strange in Go. `v := hexutil.Uint64(parsedValue)` looks ordinary.

Reply:
Will fix